avocado/utils/ssh.py

```diff
@@ -155,11 +156,17 @@ def cmd(self, command):
 
         :param command: the command to execute over the SSH session
         :param command: str
```

Comment:
If possible, also change this documentation as it should be `:type command: str`.